### ISSUE TITLE
A11y: Add main landmark to example site

### ISSUE THREAD

**Issue by arcepaul** [NCIOCPL/ncids#726]
The ncids example site is currently missing `<main>` landmarks which gets flagged as a missing best practice when running axe dev tools.  Add in the missing main landmark on all pages of the example site.

Estimate: **1**
